### ISSUE TITLE
Error Executing `idgnn_link.py` on Rel-Trial Task

### ISSUE THREAD

**Issue by Janus117** [snap-stanford/relbench#294]
script: 
  `python idgnn_link.py --dataset rel-trial --task condition-sponsor-run --num_layers 2 --epochs 30`
output:

>  UserWarning: Did not sample a single 'sponsors' node in any mini-batch. Try to increase the number of layers/hops and re-try. If you run into memory issues with deeper nets, decrease the batch size.
Epoch: 01, Train loss: nan, ...

As I check, `num_layers=2, num_neighbors=[128, 64]` causes the **sponsor** table is not involved in the batch. Please, provide the correct hyperparameter settings.

**Comment by rishabh-ranjan** [comment]
The hyperparameter details for rel-trial are available in appendix C.2. Please refer to them.

<img width="1093" alt="Image" src="https://github.com/user-attachments/assets/0c733521-15c2-4574-8e2e-f846e8c20505" />
